appinfo/routes/routesRoomController.php

```diff
@@ -100,6 +100,8 @@
 		['name' => 'Room#getParticipantByDialInPin', 'url' => '/api/{apiVersion}/room/{token}/pin/{pin}', 'verb' => 'GET', 'requirements' => array_merge($requirementsWithToken, [
 			'pin' => '\d{7,32}',
 		])],
+		/** @see \OCA\Talk\Controller\RoomController::getParticipantByDialOutNumber() */
+		['name' => 'Room#getParticipantByDialOutNumber', 'url' => '/api/{apiVersion}/room/{token}/verify-dialout', 'verb' => 'GET', 'requirements' => $requirementsWithToken],
```

Comment:
Shouldn't this be a `POST`?

Reply:
`getParticipantByDialInPin` is also a GET? `createGuestByDialIn` is a POST, but that one also creates something in server (a guest), while `getParticipantByDialOutNumber` just returns something.

Reply:
Well, I was wondering if you want all the parameters in the URL query (and with that in the webserver logs). For privacy reasons it should rather be a `POST` in the  `getParticipantByDialInPin` to not have the PINs in the logs.

But if you want a `GET`, it's fine for me. How should I encode the `options` array for the query parameter? As JSON string?

Reply:
> JSON string

Yeah, that would be best for Talk side

Reply:
> Well, I was wondering if you want all the parameters in the URL query (and with that in the webserver logs). For privacy reasons it should rather be a POST in the getParticipantByDialInPin to not have the PINs in the logs.

Makes sense. We can introduce a POST in parallel for the existing endpoint and deprecate the GET for a future release, so we are not breaking directly?

Reply:
> Makes sense. We can introduce a POST in parallel for the existing endpoint and deprecate the GET for a future release, so we are not breaking directly?

:+1: but we will need a capability so clients know they can use the `POST` endpoint.

Reply:
Capability `sip-support-dialout` added and slightly adjusted the URLs as well
- `POST /room/{token}/verify-dialin` - To replace the existing one for checking a PIN
- `POST /room/{token}/verify-dialout` - To replace this new one for checking a dialout number